client/components/quizSelection/styles.js

```diff
@@ -2,16 +2,6 @@ import styled from "styled-components";
 import { breakpoint } from "../../frontend-config";
 import { PrimaryButton, riseUp } from "../shared/styles";
 
-export const QuizzesHeader = styled.section`
```

Comment:
Hmm, was this never used? .. I didn't find it anywhere else in the project 🙂

Reply:
@Soreg this was used on the QuizSelection page when I wrote it, it has been removed in place of the common styling used by the hew shared PageHeader component.